### DISCUSSION TITLE
[FEATURE] Add a mob processor similar to the cloche

### DISCUSSION THREAD

**Discussion by Buuz135** [InnovativeOnlineIndustries/Industrial-Foregoing#138] [Feature Requests]
Industrial Foregoing Version: 
TeslaCoreLib Version: 
Crashlog (If Any):

**Comment by OneEyeMaker** [comment]
Cool feature!
But be aware: by doing this you make useless such mods, like [Woot](https://github.com/Ipsis/Woot).
IMHO, it's better to add some kind of integration with such mods rather then redo functionality of ones on your own.

**Comment by ghost** [comment]
There are lots of duplicated machines in MC. Since pack-makers can turn it off, I'm good with having an alternative. Woot is quite the undertaking and provided with any alternative, many will just make a mob farm.
